rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceTest.scala

```diff
@@ -683,7 +684,7 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
 
     an[ReduceError] should be thrownBy {
       val result = withTestStore { store =>
-        val reducer      = RholangOnlyDispatcher.create(store).reducer
+        val reducer      = RholangOnlyDispatcher.create[Task, Task.Par](store).reducer
```

Comment:
I think you could use `[Id, Id]` as I did in the PR against your repository.

Reply:
I agree that you *could*, but that's not how the tests were written and I think it's less invasive on my end to keep them as is.  If you'd like to change the tests in another PR, that makes sense to me.

Reply:
OK, fair enough.